api/envoy/extensions/filters/http/ip_tagging/v3/ip_tagging.proto

```diff
@@ -51,11 +53,27 @@ message IPTagging {
     repeated config.core.v3.CidrRange ip_list = 2;
   }
 
+  // Specifies the content of the IP tag file
+  //
+  // Allow the file to be created with no IP tags.
+  message IPTagFile {
+    repeated IPTag ip_tags = 1;
+  }
+
   // The type of request the filter should apply to.
   RequestType request_type = 1 [(validate.rules).enum = {defined_only: true}];
 
+  // Specifies the path to the IP tags file from filesystem.
```

Comment:
What happens if I specify both `path` and `ip_tags` and there's a repeated tag? Which one takes precedence?

Reply:
What happens if the file isn't found or we fail to parse/load it? Do we give up? Do we silently ignore?

Reply:
Does the path have to be absolute?

Reply:
Also: add comment from below, about the file format (can be JSON or YAML)